File: client/python/AI.py
#-*-python-*-
from BaseAI import BaseAI
from GameObject import *

class AI(BaseAI):
    """The class implementing gameplay logic."""
    @staticmethod
    def username():
        return "Shell AI"

    @staticmethod
    def password():
        return "password"

    #This function is called once, before your first turn
    def init(self):
        self.sendTag("shell")
	self.sendTag("version 0")

    #This function is called each time it is your turn
    #Return true to end your turn, return false to ask the server for updated information
    def run(self):
        return 1


      

File: client/c/AI.cpp
#include "AI.h"

/// The user name with which you logged into this computer
const char* AI::username()
{
  return "Shell AI";
}

/// The password with which you logged into this computer
const char* AI::password()
{
  return "password";
}

/// This function is run once, before your first turn.
void AI::init()
{
  sendTag("shellAI");
  sendTag("version 0");
}

/// This function is called each time it is your turn.

/// Return true to end your turn, return false to ask the server for updated information.
///
bool AI::run()
{
  return true;
}
